v03_pipeline/lib/annotations/sv.py

```diff
@@ -72,7 +82,7 @@ def _sv_types(ht: hl.Table) -> hl.ArrayExpression:
 
 
 def algorithms(ht: hl.Table, **_: Any) -> hl.Expression:
-    return hl.str(',').join(ht.info.ALGORITHMS)
+    return hl.str(',').join(ht['info.ALGORITHMS'])
```

Comment:
this change feels like its less readable/ maintainable than the current syntax. Whats the motivation for this change?

Reply:
selecting individual fields `info.ALGORITHMS` rather than just `info` at was substantially easier than selecting `info` and trying to type assert fields from a nested struct.

One potential option would be to select the nested fields and follow hail's default behavior of lifting the select to the top unnested level:

```
ht = ht.select(ht.info.ALGORITHMS)
ht.ALGORITHMS
```

leaving this line as:

```
hl.str(',').join(ht.ALGORITHMS)
```

I thought it was more maintainable to leave the `info` in here so that the structure of the import was preserved, but it definitely _looks_ worse.

Reply:
yeah I might wait for @jklugherz to come back to see if she has a preference, since both options have their cons. Personally i feel like lifting things to the top level is more readabe, but thats not like an objective choice

Reply:
yep that seems reasonable!

Reply:
hmm before I can comment on this I am confused why the original syntax needs to be changed at all

Reply:
The drive behind the change was changing the initial select to be nested, so selecting `info.ALGORITHMS` when we import the callset rather than just `info`.  This leads to the fields being unnested from an inner struct at "formatting annotation" time.  

The decision here is whether or not it's cleaner and more readable to leave the `info.` prefix in the unnested annotation names or leave hail's default behavior of renaming the field to the leaf function.  The actual change is [here](https://github.com/broadinstitute/seqr-loading-pipelines/pull/797/files#diff-dea15076a8ab788d04e8e4a78eabb479d472e5f389152cea68d0adabe99e141dR166).  My preference was to leave `info` in the field names so that the imported callset would at least pseudo-match the structure of the raw callset, but think it _barely_ matters because we're changing the structure anyways.  

We have occasionally run into issues with the default behavior (e.g. `info.AF` and `AF` both being present on the old pipeline tables) but don't think that's going to be a serious issue going forwards.

Reply:
I think it's best to leave it how you have it, doing a select to lift the nested field to the top is more confusing imo